llvm/lib/MC/MCParser/AsmParser.cpp

```diff
@@ -3123,8 +3129,9 @@ bool AsmParser::parseDirectiveAscii(StringRef IDVal, bool ZeroTerminated) {
     // Only support spaces as separators for .ascii directive for now. See the
     // discusssion at https://reviews.llvm.org/D91460 for more details.
     do {
-      if (parseEscapedString(Data))
+      if (parseEscapedString(Data, /* WarnNewline */ true))
```

Comment:
The correct style is:
```suggestion
      if (parseEscapedString(Data, /*WarnNewline=*/true))
```